README.md

```diff
@@ -184,6 +210,42 @@ for i, pose in poses_df.iterrows():
     view_mat = np.linalg.inv(mat4)
 ```
 
+### Camera rig ROS JSON file
+
+Camera rig JSON files contain spacial data that uses the same coordinate system described in the `Camera pose output` section. It contains 5 fields for each rig camera:
+
+- `intrinsics` - Camera intrinsics matrix
+- `coord_sys` - Type of the coordinate system, `RDF` is required
+- `rotation` - Rotation quaternion relative to the rig origin
+- `translation` - Translation vector relative to the rig origin
+- `sensor_size` - Sensor width and height in pixels, used to calculate camera FOV
+
+Following is a ROS JSON file example for a rig with two parallel cameras with the FOV of 90 degrees, facing forward. The difference can be found in the sign of the translation vector X-axis.
+
+``` JSON
+{
+	"cameras":
+	{
+		"c0":
+		{
+			"intrinsics": [ 960, 0, 960, 0, 960, 540, 0, 0, 0 ],
+			"coord_sys": "RDF",
```

Comment:
We can omit this field since it is unused

Reply:
I thought it might be useful for clarity, will remove it